nav2_system_tests/src/dummy_controller/dummy_controller.cpp

```diff
@@ -50,7 +49,7 @@ DummyController::~DummyController()
   RCLCPP_INFO(get_logger(), "Shutting down DummyController");
 }
 
-TaskStatus
+void
```

Comment:
Not sure I understand why TaskStatus is removed?

Reply:
TaskStatus was part of the nav2_tasks interface that I created, which has gone away. In the new Action code, the execute method void. This particular code still needs to be updasted to Actions anyways.